lib/req/steps.ex

```diff
@@ -817,6 +817,9 @@ defmodule Req.Steps do
   | zstd          | `:ezstd.decompress/1` (if [ezstd] is installed) |
   | identity      | Returns data as is                              |
 
+  This step updates the following headers to reflect the changes:
+  - `content-legnth` is set to the length of the decompressed body
+
```

Comment:
We'll have to add `content-encoding` is removed, but in another PR. Hence the list with one item for now